### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by pauldoomgov**
Adds linting and tfsec using CircleCI.

This also applies `terraform fmt` directly to the forehead.

### CODE REVIEW

File: kms_keymaker/main.tf [hunk 1]
```diff
@@ -148,11 +148,11 @@ data "aws_iam_policy_document" "kms_events_topic_policy" {
...
-#    condition {
-#      test     = "StringLike"
-#      variable = "aws:SourceArn"
-#      values = [aws_cloudwatch_event_rule.decrypt.arn]
-#    }
+    #    condition {
+    #      test     = "StringLike"
+    #      variable = "aws:SourceArn"
+    #      values = [aws_cloudwatch_event_rule.decrypt.arn]
+    #    }
```
Comment (zachmargolis) [inline kms_keymaker/main.tf:155]: lol @ formatting comments correctly

File: s3_bucket_block/main.tf [hunk 2]
```diff
@@ -100,7 +100,7 @@ module "bucket_config" {
 # -- Outputs --
 output "buckets" {
   description = "Map of the bucket names:ids created from bucket_data."
-  value       = zipmap(
-      sort(keys(aws_s3_bucket.bucket)[*]),
-      sort(values(aws_s3_bucket.bucket)[*]["id"]))
+  value = zipmap(
+    sort(keys(aws_s3_bucket.bucket)[*]),
+  sort(values(aws_s3_bucket.bucket)[*]["id"]))
```
Comment (zachmargolis) [inline s3_bucket_block/main.tf:105]: very curious how this got unindented :/

```suggestion
    sort(values(aws_s3_bucket.bucket)[*]["id"]))
```

this is what I'd expect

**Review by zachmargolis — APPROVED** [review]
LGTM

File: kms_keymaker/main.tf (hunk 1 above)
Comment (pauldoomgov) [reply kms_keymaker/main.tf:155]: You thought Rubocop was mean.....

**Pull request merged**